### ISSUE TITLE
WFTPL

### ISSUE THREAD

**Issue by YoloClin** [kelseyhightower/nocode#94]
I need this to be licenced under WTFPL for use in our apache-free environment, could this be relicenced please?

**Comment by a13xb** [comment]
I agree, the license needs to be changed. Right now any time I am writing no code (and hence distributing "nocode") I have to include a copy of the Apache 2.0 license. I have to carry a stack of Apache 2.0 license copies with me any time I go out of my house to give out to everyone I see. Needless to say it is often not appreciated by strangers (to put it mildly) and creates a huge inconvenience in my daily routine.

**Comment by lloeki** [comment]
I think the spirit of the WTFPL license is much closer to the ethos of this nocode project than Apache 2.0, but it could be improved even further. @kelseyhightower, time for a null license?

**Comment by racerpeter** [comment]
May I suggest NCL-1.0? https://github.com/kelseyhightower/nocode/pull/335
